source/clientsideapp.md

```diff
@@ -105,7 +105,69 @@ Also, an `index.html` file needs to be at the root of your repository with your
 
 ### Deploying the application
 
-Once we've packaged our application and did a `Hello World!` in our `index.html`, we must make it available for installation. To achieve this, we can create a [Github](https://github.com/) repository, and push our work on it.
+Once we've packaged our application and did a `Hello World!` in our `index.html`, we must make it available for installation.
+
+Thanks to our awesome team members, you've got not only one nor two, but three solutions to deploy your application in your local environment. Let's see what you can use!
+
+#### 1. Serve local files throught a proxy inside the VM
+
+This solution is dedicated to client-side only apps, and is the prefered way to test your app.
+
+In your terminal, simply go to your app directory, and run the `cozy-dev proxify` command.
```

Comment:
Hmm, yep, but the layout present code side-by-side with text, so without it, the text part becomes a non-sens…